plugins/woocommerce/includes/class-wc-order-item-product.php

```diff
@@ -14,6 +14,22 @@
  */
 class WC_Order_Item_Product extends WC_Order_Item {
 
+	/**
+	 * Legacy values.
+	 *
+	 * @deprecated 4.4.0 For legacy actions.
+	 * @var array
+	 */
+	public $legacy_values;
+
+	/**
+	 * Legacy cart item key.
+	 *
+	 * @deprecated 4.4.0 For legacy actions.
+	 * @var string
+	 */
+	public $legacy_cart_item_key;
+
```

Comment:
Should this be here?

Reply:
Could be separate PR, but yeah, there are deprecated notices on checkout reported by query monitor.